src/podman.scss

```diff
@@ -308,15 +308,15 @@
     min-height: 4rem;
 }
 
-.content-filter input[type="text"] {
-    display: inline-block;
-    width: 200px;
-    margin-left: 6px;
-    margin-top: 2px;
+.content-filter .ct-select {
```

Comment:
This is the CSS place that updates the .content-filter class, but the previous commit removes it.

Reply:
Yep, this styles the remaining content-filter, instead of _both_. :wink:

Reply:
There were some other duplicate-nested issues elsewhere, and I removed those as well.

Reply:
Specifically, https://github.com/cockpit-project/cockpit-podman/pull/148/commits/3a520f0a0dc33f41639b3f7ff5303e5f9e5a5e43 and https://github.com/cockpit-project/cockpit-podman/pull/148/commits/343c917fa06baf3b0bdf4dc5b7910784bb0ff8e4 (I kept container-fluid in the parents of each).